lms/static/sass/shared-v2/_variables.scss

```diff
@@ -0,0 +1,7 @@
+// LMS variables
+
+$lms-background-color: palette(grayscale, white);  // Why is the Pattern Library default background different?
```

Comment:
@AlasdairSwan @chris-mike @clrux Is there a reason why we have a slight off-white background versus a pure ``#fff``? Is there a definition somewhere of the colors we should be using?

BTW, here's the PR where these Pattern Library constants were defined:

https://github.com/edx/ux-pattern-library/pull/233